hubstorage/resourcetype.py

```diff
@@ -26,7 +28,9 @@ def _iter_lines(self, _path, **kwargs):
 
         r = self.client.request(**kwargs)
 
-        return r.iter_lines()
+        lines = r.iter_lines()
+        return (l.decode('utf8') if isinstance(l, six.binary_type)
+                else l for l in lines)
```

Comment:
The reason I'm asking is because it is not internal change - it changes output for public `iter_json` methods:

in master

```
In [5]: list(job.items.iter_json())
Out[5]:
['{"_key":"2222886/1/103/0","uuid":0}',
...
```

in py3 branch (Python 2.7.10)

```
In [6]: list(job.items.iter_json())
Out[6]:
[u'{"_key":"2222886/1/103/0","uuid":0}',
...
```

There could be cases where users rely on `iter_json` output to be strings in Python 2. Don't you think we need to keep backwards compatibility in Py2? 

I'm okay to keep it simple in Python3 - no need to output bytes, method says it iterates through data as strings.